pyaedt/hfss.py

```diff
@@ -840,7 +840,24 @@ def create_voltage_source_from_objects(self, startobj, endobject, axisdir=0, sou
         str
            Name of source created when successful, ``False`` otherwise.
 
+        Examples
+        --------
+
+        Create two boxes that will be used to create a voltage source
+        named ``'VoltageSource'``.
+
+        >>> box1 = hfss.modeler.primitives.create_box([30, 0, 0], [40, 10, 5],
+        ...                                           "BoxVolt1", "copper")
+        >>> box2 = hfss.modeler.primitives.create_box([30, 0, 10], [40, 10, 5],
+        ...                                           "BoxVolt2", "copper")
+        >>> hfss.create_voltage_source_from_objects("BoxVolt1", "BoxVolt2",
+        ...                                                  hfss.AxisDir.XNeg,
+        ...                                                  "VoltageSource")
```

Comment:
```suggestion
        >>> hfss.create_voltage_source_from_objects("BoxVolt1", "BoxVolt2",
        ...                                          hfss.AxisDir.XNeg,
        ...                                          "VoltageSource")
```
White space suggestion.